### DISCUSSION TITLE
Splash screen for android

### DISCUSSION THREAD

**Discussion by lifof** [cagnulein/qdomyos-zwift#353] [Ideas]
Just a suggestion if there's a splash screen available we could include it on Android.

**Comment by cagnulein** [comment]
For ios it comes automatically.
If you want to add it, you're welcome!

Il dom 6 giu 2021, 19:51 lifof ***@***.***> ha scritto:

**Reply by lifof** [reply]
Is there an already made one ??

**Comment by cagnulein** [comment]
Nope i never did one.
On iOS i just used the icon over a black background

Il dom 6 giu 2021, 20:14 lifof ***@***.***> ha scritto:

**Reply by lifof** [reply]
OK will see what I can do after testing and merging codeRefactoring -> master

Have nice day
